packages/pf4-component-mapper/demo/demo-schemas/sandbox.js

```diff
@@ -282,11 +284,13 @@ const output = {
                   title: 'Dropdown multiselect',
                   dataType: 'string',
                   component: components.SELECT_COMPONENT,
+                  isSearchable: true,
                   multi: true,
+                  simpleValue: false,
```

Comment:
Can you please document this prop in demo-page? :pencil2:  :book:

Reply:
doune